em_examples/TDEMHorizontalLoopCylWidget.py

```diff
@@ -0,0 +1,775 @@
+                    val = self.Ey
+                else:
+                    # ax.imshow(self.im)
+                    ax.set_xticks([])
+                    ax.set_yticks([])
+                    return "Dude, think twice ... only Ey for VMD"
+
+            elif Field == "J":
+                label = "Current density (A/m$^2$)"
+                if view == "y":
+                    val = self.Jy
+                else:
+                    # ax.imshow(self.im)
+                    ax.set_xticks([])
+                    ax.set_yticks([])
+                    return "Dude, think twice ... only Jy for VMD"
+
+            out = Utils.plot2Ddata(
+                self.mesh2D.gridCC, val, vec=vec, ax=ax,
+                contourOpts={"cmap": "viridis"},
+                ncontour=200, scale=scale
+            )
+            if scale == "linear":
+                cb = plt.colorbar(
+                    out[0], ax=ax, ticks=np.linspace(val.min(), val.max(), 3),
+                    format="%.1e"
+                )
+            elif scale == "log":
+                cb = plt.colorbar(
+                    out[0], ax=ax, ticks=np.linspace(val.min(), val.max(), 3),
+                    format="$10^{%.1f}$"
+                )
+            else:
+                raise Exception("We consdier only linear and log scale!")
+            cb.set_label(label)
+            xmax = self.mesh2D.gridCC[:, 0].max()
+            if Geometry:
+                if Scenario is 'Layer':
+                    ax.plot(
+                        np.r_[-xmax, xmax], np.ones(2)*self.srcLoc[2], 'w-', lw=1
+                    )
+                    ax.plot(np.r_[-xmax, xmax], np.ones(2)*self.z0, 'w--', lw=1)
+                    ax.plot(np.r_[-xmax, xmax], np.ones(2)*self.z1, 'w--', lw=1)
+                    ax.plot(np.r_[-xmax, xmax], np.ones(2)*self.z2, 'w--', lw=1)
+                    ax.plot(0, self.srcLoc[2], 'ko', ms=4)
+                    ax.plot(self.rxLoc[0, 0], self.srcLoc[2], 'ro', ms=4)
+                elif Scenario is 'Sphere':
+                    ax.plot(
+                        np.r_[-xmax, xmax], np.ones(2)*self.srcLoc[2], 'k-', lw=1
+                    )
+                    ax.plot(np.r_[-xmax, xmax], np.ones(2)*self.z0, 'w--', lw=1)
+                    ax.plot(np.r_[-xmax, xmax], np.ones(2)*self.z1, 'w--', lw=1)
+                    ax.plot(np.r_[-xmax, xmax], np.ones(2)*(self.z1-self.h), 'w--', lw=1)
+                    Phi = np.linspace(0, 2*np.pi, 41)
+                    ax.plot(self.R*np.cos(Phi), self.z2+self.R*np.sin(Phi), 'w--', lw=1)
+                    ax.plot(0, self.srcLoc[2], 'ko', ms=4)
+                    ax.plot(self.rxLoc[0, 0], self.srcLoc[2], 'ro', ms=4)
+
+            ax.set_xlabel("Distance (m)")
+            ax.set_ylabel("Depth (m)")
+            ax.set_title(title)
+            ax.text(
+                -150, 150, ("Time: %.3f ms") % (self.prb.times[itime]*1e3),
+                fontsize=16, fontweight='bold', color='r'
```

Comment:
This is really hard to see. What about just including it in the title?
```python
title = title + "\nTime: %.3f ms" % (self.prb.times[itime]*1e3)
```

Reply:
e.g. it will look like (for FDEM)
![image](https://user-images.githubusercontent.com/6361812/42847654-7348eb72-89d1-11e8-977e-4f2b9f937770.png)